contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 5 A/B testing mistakes and how to avoid them
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: todo
+tags: ['experimentation']
+---
```

Comment:
```suggestion
---
date: 2023-04-24
title: "5 A/B testing mistakes and how to avoid them"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ['lior-neu-ner']
featuredImage: todo
featuredImageType: full
category: Product growth
tags: 
  - Guides
---

Reply:
This is the correct frontmatter for the blog